package.json

```diff
@@ -18,6 +18,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
Did we need to add `react-router-dom`

Reply:
No for this I was working on another project and install to the wrong terminal I think